documentation/book/ref-sample-kafka-resource-config.adoc

```diff
@@ -0,0 +1,189 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-bridge-configuration.adoc
+
+[id='ref-kafka-bridge-producer-configuration-{context}']
+= Sample Kafka YAML configuration
+
+For help in understanding the configuration options available for your Kafka deployment, refer to sample YAML file provided here.
+
+[source,shell,subs="+quotes,attributes"]
+
+----
+apiVersion: kafka.strimzi.io/v1beta1
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    replicas: 3 <1>
+    version: {ProductVersion} <2>
+    resources: <3>
+      requests: <4>
+        memory: 64Gi
+        cpu: "8"
+      limits: <5>
+        memory: 64Gi
+        cpu: "12"
+    jvmOptions: <6>
+      -Xms: 8192m
+      -Xmx: 8192m
+    listeners: <7>
+      tls: <8>
+        authentication:<9>
+          type: tls
+      external: <10>
+        type: route
+        authentication:
+          type: tls
+    authorization: <11>
+      type: simple
+    config: <12>
+      auto.create.topics.enable: "false"
+      offsets.topic.replication.factor: 3
+      transaction.state.log.replication.factor: 3
+      transaction.state.log.min.isr: 2
+    storage: <13>
+      type: persistent-claim <14>
+      size: 10000Gi <15>
+    rack:
+      topologyKey: failure-domain.beta.kubernetes.io/zone
+    metrics: <16>
+      lowercaseOutputName: true
+      rules: <17>
+      # Special cases and very specific rules
+      - pattern : kafka.server<type=(.+), name=(.+), clientId=(.+), topic=(.+), partition=(.*)><>Value
```

Comment:
Note because of `subs="+quotes..."` above the * in the regex here means we render some of this as bold.